### PR TITLE
Add binding to calculate distance between two geometries

### PR DESCRIPTION
**Pull request by gahjelle**
See #17 

Adds `distance()` measurement. The distance is calculated in meters per default, based on a new `EARTH_RADIUS_METERS` constant. The radius can also be customized in the call to `distance()`.

### CODE REVIEW

File: src/accessors-geog.cpp [hunk 1]
```diff
@@ -69,4 +79,23 @@ void init_accessors(py::module& m) {
...
+          R"pbdoc(
+        Calculate the distance between two geographies.
+
+        Parameters
+        ----------
+        a : :py:class:`Geography` or array_like
+            Geography object
+        b : :py:class:`Geography` or array_like
+            Geography object
+        radius : float
```
Comment (benbovy) [inline src/accessors-geog.cpp:97]:
```suggestion
        radius : float, optional
```

**Review by benbovy — APPROVED** [review]
Thanks @gahjelle ! Just one nit-picking suggestion for the docstrings otherwise it all looks good!

**Comment by gahjelle** [comment]
Thanks @benbovy I've updated the docstring.

File: tests/test_accessors.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import numpy as np
+import pytest
 
 import spherely
 
-import pytest
+earth_radius_meters = 6_371_010
```
Comment (jorisvandenbossche) [inline tests/test_accessors.py:6]: This is not actually used?

Comment (gahjelle) [reply tests/test_accessors.py]: Ah, good catch @jorisvandenbossche 

Thanks, I've removed it.

**Comment by benbovy** [comment]
Many thanks @gahjelle !

**Pull request merged**